### ISSUE TITLE
Entity Linking Result？

### ISSUE THREAD

**Issue by dalek-who** [studio-ousia/luke#63]
Hi authors,
I notice that 
> **Global Entity Disambiguation with Pretrained Contextualized Embeddings of Words and Entities**
> https://arxiv.org/abs/1909.00426

is also your work, and that one is nearly the same as LUKE except several slight difference. It is also awesome and is Entity Linking SOTA, but why don't you show the Entity Linking perfromance in LUKE?

**Comment by ikuyamada** [comment]
Hi,
Thank you for raising the issue.
The code used in the entity linking paper is also based on this repository. We plan to release the trained checkpoints in the future.

**Comment by dalek-who** [comment]
Thanks, waiting for your checkpoint. Can current checkpoint reproduce that linking result? I want to try it in my system.
Also, I have two question about that entity linking paper:
1. In the "5 Experiment" chapter's second paragraph, ```We mask 90% of the mentions and ﬁx the entity token embeddings (B and B ∗ ) and the output bias (b oand b o∗ ).```，how to understand ```mask 90% of the mentions```? Why not use all mentions to fineturn entity link?
2. That entity linking pretrain takes 8 V100* 10 days, but LUKE takes 16 V100* 30days, why?

**Comment by ikuyamada** [comment]
We did not test the performance of entity linking using the current checkpoint.

> In the "5 Experiment" chapter's second paragraph, We mask 90% of the mentions and ﬁx the entity token embeddings (B and B ∗ ) and the output bias (b oand b o∗ ).，how to understand mask 90% of the mentions? Why not use all mentions to fineturn entity link?

Since the paper uses *global* context based on the entity sequence to disambiguate mentions, it is required to train the model to use the non-masked entities as contexts. If we mask 100% of mentions, the model cannot learn such interactions during fine-tuning.

> That entity linking pretrain takes 8 V100* 10 days, but LUKE takes 16 V100* 30days, why?

We used different servers with different training steps and batch size :) Please note that this repository is not the official code of the entity linking paper.

**Comment by dalek-who** [comment]
Thanks!

**Comment by ikuyamada** [comment]
Hi @dalek-who,
I've just released the code and model checkpoint file [here](https://github.com/studio-ousia/luke/tree/master/examples/entity_disambiguation).

**Comment by dalek-who** [comment]
Thank you for this excellent work !!
